quay-enterprise/running-chihaya.md

```diff
@@ -0,0 +1,69 @@
+# Chihaya Setup
+
+The <a href="https://github.com/chihaya/chihaya">Chihaya</a> project is an open source torrent tracker that supports JWT-based authorization. It is the preferred tracker for making use of the secure [BitTorrent-based distribution](bittorrent.md) feature in Quay Enterprise.
+
+## Initial Setup
+
+### Basic configuration
+
+Copy the following file as `config.json` to an empty directory, replacing `{QE LOCATION}` and `{TRACKER LOCATION}` with
+the reachable endpoint for the Quay Enterprise instance and the tracker itself, respectively.
+
+```
+{
+  "createOnAnnounce": true,
+  "purgeInactiveTorrents": true,
+  "announce": "15m",
+  "minAnnounce": "15m",
+  "reapInterval": "60s",
+  "reapRatio": 1.25,
+  "defaultNumWant": 50,
+  "torrentMapShards": 16,
+  "allowIPSpoofing": true,
+  "dualStackedPeers": true,
+  "realIPHeader": "X-Forwarded-For",
+  "respectAF": false,
+  "clientWhitelistEnabled": false,
+  "clientWhitelist": ["OP1011"],
+  "apiListenAddr": "0.0.0.0:6880",
+  "apiRequestTimeout": "4s",
+  "apiReadTimeout": "4s",
+  "apiWriteTimeout": "4s",
+  "apiListenLimit": 0,
+  "udpListenAddr": "0.0.0.0:6881",
+  "httpListenAddr": "0.0.0.0:6881",
+  "httpRequestTimeout": "4s",
+  "httpReadTimeout": "4s",
+  "httpWriteTimeout": "4s",
+  "httpListenLimit": 0,
+  "driver": "noop",
+  "statsBufferSize": 0,
+  "includeMemStats": true,
+  "verboseMemStats": false,
+  "memStatsInterval": "5s",
+  "jwkSetURI": "https://{QE LOCATION}/keys/services/quay/keys",
+  "jwkUpdateInterval": "60s",
+  "jwtAudience": "http://{TRACKER LOCATION}/announce",
+  "jwkTTL": "5m"
+}
+```
+
+
+## Running
+
+Replace (or set) the environment variable `PATH_TO_DIR_WITH_CONF_FILE` to the path of the directory containing the configuration file created in the previous step. Once complete, run the following commands to start Chihaya under a Docker container with the specified configuration mounted.
+
+```sh
+$ docker pull quay.io/jzelinskie/chihaya:v1.0.0
+$ export CHIHAYA_LOG_LEVEL=5 # most verbose, and the default
+$ docker run -p 6880-6882:6880-6882 -v $PATH_TO_DIR_WITH_CONF_FILE:/config:ro -e quay.io/jzelinskie/chihaya:v1.0.0 -v=$CHIHAYA_LOG_LEVEL
```

Comment:
You probably copied & pasted this -- there's an extra `-e` that should either be a `-d` or not present. I need to fix this upstream.